Upstream-Status: Inappropriate [embedded specific]

Avoid these warning-errors

automake: warnings are treated as errors
Makefile.am:15: warning: '%'-style pattern rules are a GNU make extension
Makefile.am:21: warning: pkgconfig_DATA:-$(API_VERSION: non-POSIX variable name
addressbook/backends/file/Makefile.am: installing './depcomp'
addressbook/libebook/Makefile.am:55: warning: '%'-style pattern rules are a GNU make extension
addressbook/libebook/Makefile.am:65: warning: '%'-style pattern rules are a GNU make extension
addressbook/libebook/Makefile.am:66: warning: subst -,_,$*: non-POSIX variable name
addressbook/libebook/Makefile.am:66: (probably a GNU make extension)
addressbook/libebook/Makefile.am:72: warning: pkgconfig_DATA:-$(API_VERSION: non-POSIX variable name
addressbook/libedata-book/Makefile.am:2: warning: '%'-style pattern rules are a GNU make extension
addressbook/libedata-book/Makefile.am:3: warning: subst -,_,$*: non-POSIX variable name
addressbook/libedata-book/Makefile.am:3: (probably a GNU make extension)
glib-gen.mak:9: warning: '%'-style pattern rules are a GNU make extension
addressbook/libedata-book/Makefile.am:5:   'glib-gen.mak' included from here
glib-gen.mak:13: warning: '%'-style pattern rules are a GNU make extension
addressbook/libedata-book/Makefile.am:5:   'glib-gen.mak' included from here
glib-gen.mak:18: warning: '%'-style pattern rules are a GNU make extension
addressbook/libedata-book/Makefile.am:5:   'glib-gen.mak' included from here
glib-gen.mak:26: warning: '%'-style pattern rules are a GNU make extension
addressbook/libedata-book/Makefile.am:5:   'glib-gen.mak' included from here
glib-gen.mak:38: warning: '%'-style pattern rules are a GNU make extension
addressbook/libedata-book/Makefile.am:5:   'glib-gen.mak' included from here
glib-gen.mak:41: warning: '%'-style pattern rules are a GNU make extension
addressbook/libedata-book/Makefile.am:5:   'glib-gen.mak' included from here
addressbook/libedata-book/Makefile.am:86: warning: '%'-style pattern rules are a GNU make extension
addressbook/libedata-book/Makefile.am:101: warning: pkgconfig_DATA:-$(API_VERSION: non-POSIX variable name
calendar/backends/weather/Makefile.am:44: warning: '%'-style pattern rules are a GNU make extension
calendar/libecal/Makefile.am:53: warning: '%'-style pattern rules are a GNU make extension
calendar/libecal/Makefile.am:61: warning: '%'-style pattern rules are a GNU make extension
calendar/libecal/Makefile.am:62: warning: subst -,_,$*: non-POSIX variable name
calendar/libecal/Makefile.am:62: (probably a GNU make extension) calendar/libecal/Makefile.am:68: warning: pkgconfig_DATA:-$(API_VERSION: non-POSIX variable name
calendar/libedata-cal/Makefile.am:5: warning: '%'-style pattern rules are a GNU make extension
calendar/libedata-cal/Makefile.am:6: warning: subst -,_,$*: non-POSIX variable name
calendar/libedata-cal/Makefile.am:6: (probably a GNU make extension)
glib-gen.mak:9: warning: '%'-style pattern rules are a GNU make extension
calendar/libedata-cal/Makefile.am:8:   'glib-gen.mak' included from here
glib-gen.mak:13: warning: '%'-style pattern rules are a GNU make extension
calendar/libedata-cal/Makefile.am:8:   'glib-gen.mak' included from here
glib-gen.mak:18: warning: '%'-style pattern rules are a GNU make extension
calendar/libedata-cal/Makefile.am:8:   'glib-gen.mak' included from here
glib-gen.mak:26: warning: '%'-style pattern rules are a GNU make extension
calendar/libedata-cal/Makefile.am:8:   'glib-gen.mak' included from here
glib-gen.mak:38: warning: '%'-style pattern rules are a GNU make extension
calendar/libedata-cal/Makefile.am:8:   'glib-gen.mak' included from here
glib-gen.mak:41: warning: '%'-style pattern rules are a GNU make extension
calendar/libedata-cal/Makefile.am:8:   'glib-gen.mak' included from here
calendar/libedata-cal/Makefile.am:71: warning: '%'-style pattern rules are a GNU make extension
calendar/libedata-cal/Makefile.am:86: warning: pkgconfig_DATA:-$(API_VERSION: non-POSIX variable name
camel/Makefile.am:17: warning: '%'-style pattern rules are a GNU make extension
libebackend/Makefile.am:35: warning: '%'-style pattern rules are a GNU make extension
libebackend/Makefile.am:41: warning: pkgconfig_DATA:-$(API_VERSION: non-POSIX variable name
libedataserver/Makefile.am:91: warning: '%'-style pattern rules are a GNU make extension
libedataserver/Makefile.am:97: warning: pkgconfig_DATA:-$(API_VERSION: non-POSIX variable name
libedataserverui/Makefile.am:108: warning: '%'-style pattern rules are a GNU make extension
libedataserverui/Makefile.am:118: warning: pkgconfig_DATA:-$(API_VERSION: non-POSIX variable name
servers/google/libgdata-google/Makefile.am:27: warning: '%'-style pattern rules are a GNU make extension
servers/google/libgdata-google/Makefile.am:33: warning: pkgconfig_DATA:-$(API_VERSION: non-POSIX variable name
servers/google/libgdata/Makefile.am:31: warning: '%'-style pattern rules are a GNU make extension
servers/google/libgdata/Makefile.am:37: warning: pkgconfig_DATA:-$(API_VERSION: non-POSIX variable name
servers/groupwise/Makefile.am:85: warning: '%'-style pattern rules are a GNU make extension
servers/groupwise/Makefile.am:91: warning: pkgconfig_DATA:-$(API_VERSION: non-POSIX variable name
autoreconf: automake failed with exit status: 1

Signed-Off-By: Nitin A Kamble <nitin.a.kamble@intel.com>
2012/07/09

Index: git/configure.ac
===================================================================
--- git.orig/configure.ac
+++ git/configure.ac
@@ -21,7 +21,7 @@ m4_define([api_version], [1.2])
 dnl Autoconf / Automake Initialization
 AC_PREREQ(2.58)
 AC_INIT([evolution-data-server],[eds_version],[http://bugzilla.gnome.org/enter_bug.cgi?product=Evolution-Data-Server])
-AM_INIT_AUTOMAKE([gnu 1.9 tar-ustar -Wall -Werror foreign])
+AM_INIT_AUTOMAKE([gnu 1.9 tar-ustar -Wall foreign])
 AC_CONFIG_MACRO_DIR([m4])
 AC_CONFIG_SRCDIR(README)
 AC_CONFIG_HEADERS(config.h)
